examples/3d/bloom_3d.rs

```diff
@@ -41,15 +41,15 @@ fn setup_scene(
     ));
 
     let material_emissive1 = materials.add(StandardMaterial {
-        emissive: Color::linear_rgb(13.99, 5.32, 2.0), // 4. Put something bright in a dark environment to see the effect
+        emissive: LinearRgba::rgb(23000.0, 9000.0, 3000.0), // 4. Put something bright in a dark environment to see the effect
```

Comment:
I think these values need to be adjusted back down.

Reply:
Nice catch, this is a merging error from my side! Now resolved!